OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -132,6 +447,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">10</dim>
                                         <dim type="left">9</dim>
                                     </dimensions>
+                                    <note>The characters per line are 13-16.</note>
+                                    <ab type="ruling" subtype="pattern">
+                                        <locus from="6ra" to="53vb"/>    
```

Comment:
The range of folia is correct here? It is the same as above. You would like to provide information on three different layous, right?